srcpkgs/kodi-platform/template

```diff
@@ -3,15 +3,31 @@ pkgname=kodi-platform
 version=17.1.0
 revision=1
 build_style=cmake
-configure_args="-DCMAKE_INSTALL_LIBDIR=/usr/lib"
+configure_args="-DCMAKE_INSTALL_LIBDIR=/usr/lib
+ -DCMAKE_MODULE_PATH=${XBPS_CROSS_BASE}/usr/share/kodi/cmake"
 makedepends="kodi-devel tinyxml-devel p8-platform-devel"
 short_desc="Kodi platform support library"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-or-later"
 homepage="https://github.com/xbmc/kodi-platform"
 distfiles="https://github.com/xbmc/kodi-platform/archive/${version}.tar.gz"
 checksum=39de7eb6db14de3fa33ca0e56243c3307e7598187e73ad930dabccdf068bd9b2
-nocross="kodi cmake files use absolute paths"
+
+post_build() {
+	# Remove cross-cruft
+	if [ "$CROSS_BUILD" ]; then
+		vsed -e '/libdir/s,=.*,=/usr/lib,' \
+			-e '/CXX/s,=.*,=c++,' \
+			-e '/CC/s,=.*,=gcc,' \
+			-e "s,${XBPS_CROSS_BASE},,g" \
+			-i build/kodiplatform.pc
+
+		vsed -e 's,/usr//,/usr/,g' \
+			-e "s,${XBPS_CROSS_BASE},,g" \
+			-e 's,/usr/usr,/usr,g' \
```

Comment:
Given that the file looks like

```
prefix=/usr
libdir=/usr//usr/lib
includedir=/usr/include
CXX=/usr/lib/ccache/bin/g++ 
CC=/usr/lib/ccache/bin/cc 
DEPENDENCIES=/usr/lib/libtinyxml.so

Name: kodiplatform
Description: Kodi add-on platform support library 18.0
Version: 18.0
Libs: -L -l:kodiplatform -ltinyxml
Cflags: -fstack-clash-protection -D_FORTIFY_SOURCE=2 -mtune=generic -O2 -pipe   -g -std=c++11 -fPIC
```

for native builds, the issue is that `/usr//usr/lib` appears at all. I would attribute it to `-DCMAKE_INSTALL_LIBDIR=/usr/lib`.